webrtc.html

```diff
@@ -5396,10 +5396,87 @@ <h3 id="sec.identity-verify-assertion">Verifying Identity Assertions</h3>
     </table>
   </section>
 
-  <section>
+  <section class="informative">
     <h2>Security Considerations</h2>
+    <p>This section is non-normative; it specifies no new behaviour,
+    but instead summarizes information already present in other parts
+    of the specification.</p>
+
+    <p>This document extends the Web platform with the ability to set
+    up real time, direct communication between browsers and other
+    devices, including other browsers.</p>
+
+    <p>This means that data and media can be shared between
+    applications running in different browsers, or between an
+    application running in the same browser and something that is not a
+    browser, something that is an extension to the usual barriers in
+    the Web model against sending data between entities with different
+    origins.</p>
+
+    <p>The WebRTC specification provides no user prompts or chrome
+    indicators for communication; it assumes that once the Web page
+    has been allowed to access media, it is free to share that media
+    with other entities as it chooses.</p>
```

Comment:
Might be worth mentioning there are modes where the web page does not get access to the data but it is still sets up an audio / video stream